go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/crossplane/crossplane-tools
 go 1.12
 
 require (
-	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
-	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
+	github.com/crossplane/crossplane-runtime v0.9.0 // indirect
```

Comment:
Do we need this? Is it to keep versions of other dependencies same with crossplane-runtime?

Reply:
No, this is an unintentional inclusion. I haven't got to the bottom of it yet, but my Go install is insistent on making seemingly unnecessary module changes when I run `make build` lately.